va/hci/edpm-post-ceph/nodeset/kustomization.yaml

```diff
@@ -77,6 +77,28 @@ replacements:
           - spec.glance.template.glanceAPIs.default.replicas
         options:
           create: true
+  - source:
+      kind: ConfigMap
+      name: service-values
+      fieldPath: data.cinderAPI.replicas
```

Comment:
This all looks fine, but the rest of the cinder replacements are above L58. Here you've added these cinder replacements after glance stuff. Would it make more sense to group the cinder replacements together?

Reply:
That would make sense to me too, but otherwise this looks good

Reply:
OK, I've done that.